packages/outline-view/src/browser/outline-view-contribution.ts

```diff
@@ -32,7 +32,8 @@ export class OutlineViewContribution extends AbstractViewContribution<OutlineVie
                 area: 'right',
                 rank: 500
             },
-            toggleCommandId: 'outlineView:toggle'
+            toggleCommandId: 'outlineView:toggle',
+            toggleKeybinding: 'ctrlcmd+shift+o'
```

Comment:
that's a keybinding for `Go to Symbol`

please revert it, we fixed it not so long time ago: https://github.com/theia-ide/theia/issues/2154

also please make sure for all introduced keybindings that they don't break some existing keybindings

Reply:
thanks and sorry about that, I'll make the appropriate changes